extensions/wikia/AdEngine/js/slot/floatingMedrec.js

```diff
@@ -86,14 +86,14 @@ define('ext.wikia.adEngine.slot.floatingMedrec', [
 				win.removeEventListener('scroll', update);
 				win.removeEventListener('resize', update);
 
-				$adSlot.css({
-					visibility: 'hidden'
-				});
+				// $adSlot.css({
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACommentedCode)